study.md

```diff
@@ -21,7 +21,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
+700.to_s.length
```

Comment:
This will return `3`.  We want to reverse the string so the invocation of `length` should be `reverse`.